src/components/Gradebook/index.jsx

```diff
@@ -357,64 +359,41 @@ export default class Gradebook extends React.Component {
             <h1>Gradebook</h1>
             <h3> {this.props.courseId}</h3>
             {this.props.areGradesFrozen &&
-            <div className="alert alert-warning" role="alert" >
-              The grades for this course are now frozen. Editing of grades is no longer allowed.
-            </div>
+              <div className="alert alert-warning" role="alert" >
+                The grades for this course are now frozen. Editing of grades is no longer allowed.
+              </div>
             }
             {(this.props.canUserViewGradebook === false) &&
-            <div className="alert alert-warning" role="alert" >
-              You are not authorized to view the gradebook for this course.
-            </div>
+              <div className="alert alert-warning" role="alert" >
+                You are not authorized to view the gradebook for this course.
+              </div>
             }
             <Tabs labels={this.getActiveTabs()}>
               <div>
+                <h4>Step 1: Filter the Grade Report</h4>
                 <div className="d-flex justify-content-between" >
                   {this.props.showSpinner && <div className="spinner-overlay"><Icon className={['fa', 'fa-spinner', 'fa-spin', 'fa-5x', 'color-black']} /></div>}
-
                   <div>
-                    <div role="radiogroup" aria-labelledby="score-view-group-label">
-                      <span id="score-view-group-label">Score View:</span>
-                      <span>
-                        <label className="mr-2" htmlFor="score-view-percent">
-                          <input
-                            id="score-view-percent"
-                            className="ml-2 mr-1"
-                            type="radio"
-                            name="score-view"
-                            value="percent"
-                            defaultChecked
-                            onClick={() => this.props.toggleFormat('percent')}
-                          />
-                          Percent
-                        </label>
-                      </span>
-                      <span>
-                        <label htmlFor="score-view-absolute">
-                          <input
-                            id="score-view-absolute"
-                            type="radio"
-                            name="score-view"
-                            value="absolute"
-                            className="mr-1"
-                            onClick={() => this.props.toggleFormat('absolute')}
-                          />
-                          Absolute
-                        </label>
-                      </span>
-                    </div>
+                    <InputSelect
```

Comment:
I decided to just switch this over to a dropdown to match Christine's wireframes while I was in this area.